service/service.go

```diff
@@ -410,61 +411,62 @@ func (s *Service) GetSecret(ctx context.Context, req *pbsvc.UserRequest) (*pbsvc
 // GetToken generates a token after verifying user's email and password,
 // stores generated token related info in DB, returns said token
 func (s *Service) GetToken(ctx context.Context, req *pbsvc.UserRequest) (*pbsvc.UserResponse, error) {
+	//TODO
```

Comment:
Changes like commenting out stuff can be done from the master by using `git push --force origin master`. This will lessen the noise when doing code reviews.